docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurator.java

```diff
@@ -87,6 +95,57 @@ private void configure(ArquillianDescriptor arquillianDescriptor) {
         return cubeConfiguration;
     }
 
+    private Map<String, String> resolveDownloadDockerMachine(Map<String, String> config) {
+        if (config.containsKey(CubeDockerConfiguration.DOCKER_MACHINE_NAME)) {
+            String machineVersion = null;
+            String machineArquillianPath = null;
+            File dockerMachineFile = null;
+            if (config.containsKey(CubeDockerConfiguration.DOCKER_MACHINE_VERSION)) {
+                machineVersion = config.get(CubeDockerConfiguration.DOCKER_MACHINE_VERSION);
+                machineArquillianPath = CubeDockerConfiguration.resolveMachinePath(machineVersion);
+                dockerMachineFile = new File(machineArquillianPath);
```

Comment:
I am not sure if this going to work in Windows, I think you should use something like https://github.com/arquillian/arquillian-cube/blob/master/docker/docker/src/main/java/org/arquillian/cube/docker/impl/util/HomeResolverUtil.java